images/public-log-vendor-matcher/app/ext-ip-asn.sql

```diff
@@ -0,0 +1,9 @@
+SELECT
+    asn as asn,
+    ip as cidr_ip,
+    ip_start as start_ip,
+    ip_end as end_ip,
+    NET.IPV4_TO_INT64(NET.IP_FROM_STRING(ip_start)) AS start_ip_int,
+    NET.IPV4_TO_INT64(NET.IP_FROM_STRING(ip_end)) AS end_ip_int
+    FROM `k8s-infra-ii-sandbox.${GCP_BIGQUERY_DATASET_WITH_DATE}.pyasn_ip_asn_extended`
```

Comment:
Not clear the explicit gcp project needs to be specified here, but it shouldn't be a hardcode
```suggestion
    FROM `${GCP_PROJECT}.${GCP_BIGQUERY_DATASET_WITH_DATE}.pyasn_ip_asn_extended`
```

Reply:
Ah, whoopses that shouldn't be there. Updated in d4e73426